.node-version

```diff
@@ -0,0 +1 @@
+20.17.0
```

Comment:
To have reproducibility, we need to specify the version down to the patch version, otherwise different runs might download different minor/patch versions. I don't know exactly where `.nvmrc` is used, but if we could specify this down to the patch version, then I think we could also use that file.

Reply:
Let's try that. Having it specified in two files is already too many in my opinion.

Reply:
Agreed. I will give it a try.

Reply:
It seems to work 👍